src/lib/core/testing/dispatch-events.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  createFakeEvent,
+  createKeyboardEvent,
+  createMouseEvent,
+  createTransitionEndEvent
+} from './event-objects';
+
+/** Shorthand to dispatch a fake event on a specified node. */
+export function dispatchFakeEvent(node: Node, eventName: string) {
```

Comment:
Not really sure. I thought that Fake fits pretty good. It indicates that those are not real events and can lack of event details.